src/cls/user/cls_user.cc

```diff
@@ -163,13 +163,18 @@ static int cls_user_set_buckets_info(cls_method_context_t hctx, bufferlist *in,
     CLS_LOG(20, "storing entry for key=%s size=%lld count=%lld",
             key.c_str(), (long long)update_entry.size, (long long)update_entry.count);
 
+    // Update bucket's placement rule info only when linking bucket, not on
+    // usage stats change. */
+    if (op.add) {
```

Comment:
@rzarzynski How good would it would be to place 'if(op.add)` and `if (!op.add)` in if-else statement?